### ISSUE TITLE
Oregen buff?

### ISSUE THREAD

**Issue by Rodg88** [NillerMedDild/Enigmatica6#1425]
Just to discuss, but in 0.4.0, basic (iron/copper especially) ores seem much harder to come by, possibly due to the high range of y-levels they can spawn at diluting the amount you find underground? Maybe have the ores spawn more normally, and have a separate pass for populating high y-levels so it doesn't influence the main oregen pass? Not sure if that's possible... Otherwise a frequency or vein size buff might be appropriate? Vein size might be more interesting than just frequency too.

**Comment by NillerMedDild** [comment]
Thanks for the report, this is very helpful :D
We'll look into increasing the overall amount of them, as we can't have separate configs for one ore.

**Comment by theboo** [comment]
yes, my (limited) testing was mostly done with worldstripper which can make ores look much more common that they feel in actual gameplay.. i did boost the amount but looks like not enough

**Comment by Ridanisaurus** [comment]
> Thanks for the report, this is very helpful :D
> We'll look into increasing the overall amount of them, as we can't have separate configs for one ore.

Yet. We can't have a separate config for one ore, yet. :P

**Comment by Merlo17** [comment]
Copper especially is the main offender here imo. Bringing it down 20/30 blocks would help

**Comment by TemperedFool** [comment]
I will also put a vote in for the "larger veins" suggestion. I wouldn't suggest going as far as GTCE does (yet...?), but the "hard to find but supersized and interestingly mixed" veins it does feel super rewarding imo.

Having less frequent, but much higher density veins (especially once you have some way to search for them, either PNC or maybe a reinvestigation of Scannable in #483) means less aimless digging imo.

**Comment by MuteTiefling** [comment]
seconding bigger veins. Really satisfying digging out a good 20-30 blocks of an ore

**Comment by TemperedFool** [comment]
20-30... 50-100... 100-200... those all seem like reasonable numbers...

**Comment by sciguyryan** [comment]
I would also like to add my vote in favour of this.

I have always loved the prospect of finding the "mother load" of an ore, even if I had to work to get it.

I also like the prospect of using Scannable to augment this too. I have used the mod before and it is quite fun to use.

**Comment by NillerMedDild** [comment]
I will start by buffing iron/copper/tin, as larger veins will require more testing than we have time for atm :)

**Comment by Rodg88** [comment]
I regenerated my non-base chunks in 0.4.1, definitely seeing more iron, but copper is still rare. Was mining at y50 or so, and got 5-10 copper for 60 iron. A definite improvement overall though

**Comment by Rodg88** [comment]
Tested mining at y114, definitely better, 54 iron and 50 copper from this area I hollowed out:
<img width="1577" alt="Screen Shot 2021-02-27 at 5 51 15 pm" src="https://user-images.githubusercontent.com/2611674/109378420-801c7980-7926-11eb-8381-3443959ba3ed.png">

It's better, but unless you live near some big mountains quite impractical. If they were really big veins it could be quite fun to mount a big mining expedition. But also just being able to dig under your base and reliably get copper would still be nice.
